library/SimplePie.php

```diff
@@ -2666,12 +2666,14 @@ public function get_links($rel = 'alternate')
 		if (isset($this->data['headers']['link']))
 		{
 			$link_headers = $this->data['headers']['link'];
-			if (is_string($link_headers)) {
-				$link_headers = array($link_headers);
+			if (is_array($link_headers)) {
+				$link_headers = implode(',', $link_headers);
 			}
-			$matches = preg_filter('/<([^>]+)>; rel='.preg_quote($rel).'/', '$1', $link_headers);
-			if (!empty($matches)) {
-				return $matches;
+			// https://datatracker.ietf.org/doc/html/rfc8288
+			if (is_string($link_headers) &&
```

Comment:
Can `$this->data['headers']['link']` be anything other than array or string?

Reply:
I do not believe so. And this PR does not change the assumptions (there was already a case for array and string before).

Reply:
In that case, is not `is_string($link_headers)` redundant?

Reply:
Maybe, but it is not something that is currently enforced (e.g. through a type), and I have not checked in details